### DISCUSSION TITLE
Custom domain name and SimpleLogin extension.

### DISCUSSION THREAD

**Discussion by condorfleck** [simple-login/app#1539] [Q&A]
With the premium plan and just one forwarding address, will the SimpleLogin extension generate aliases for a custom domain name?  If so, that would be a great time saver.   

Thanks in advance.

**Comment by nguyenkims** [comment]
@condorfleck yes you can choose your domain as default domain when creating a new alias. This option can be enabled on https://app.simplelogin.io/dashboard/setting
